python/MDSplus/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+    requires = ["setuptools>=44.1.1", "wheel"] # python2.7 support
+    build-backend = "setuptools.build_meta"
+
+[project]
+    name='MDSplus'
+    authors = [
+        {name = "MDSplus Development Team", email = "twf@www.mdsplus.org"},
```

Comment:
We should probably change the email here, maybe mdsplusadmin@psfc.mit.edu ?

Reply:
OK.. done.